.ci/Jenkinsfile

```
@@ -7,6 +7,9 @@ pipeline {
   environment {
     REPO = 'e2e-testing'
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    APM_ENVIRONMENT="ci"
+    ENABLE_INSTRUMENTATION="false"
```

Activate this once we are sure we are not leaking secrets
```suggestion
    ENABLE_INSTRUMENTATION="true"
```